playbooks/roles/tower/defaults/main.yml

```diff
@@ -1,10 +1,11 @@
 ---
 # Ansible Tower Configurations
 tower_login: true
-tower_host: ''
-tower_username: ''
-tower_password: ''
-tower_verify_ssl: false
+tower_environment: "{{ lookup('env','tower_environment') | default('local') }}"
```

Comment:
I've left it defaulting to "local" here for the `tower_environment` variable, perhaps we can also do some work in the future to enable default testing via minishift or something.